### PR TITLE
Update fixture files and add admission_type dimensions

### LINKED ISSUE #820: Update fixture files
**What needs to be done? Why does it need to be done?**
There are a few updates we need to make to the fixtures:
- [x] remove the `district` dimension (all files)
- [x] update the `admission_type` values to be as shown below (stand by) (all files)
 ALL 
LEGAL_REVOCATION
SHOCK_INCARCERATION_0_TO_6_MONTHS 
SHOCK_INCARCERATION_6_MONTHS      
SHOCK_INCARCERATION_9_MONTHS      
SHOCK_INCARCERATION_12_MONTHS     
SHOCK_INCARCERATION_PVC           

- [x] add the `recommended_for_revocation_count ` and `recommended_for_revocation_count_all ` columns to support the officer recommendation bar in the race and gender charts (race and gender files only)

It will most likely make sense to base these of of PA data since we are deep in the sub-office filtering/district chart work. But you will want to test the files in json format, after randomizing, and before exporting to optimized format to ensure that we have a decent set of data on the MO dashboard with `district='ALL' selected.

**Additional context**
Add any other context about the task here.

### PR DESCRIPTION
**Pull request by daschi**
## Description of the change

This PR adds the updated fixture files using US_PA data. The district distribution file has both US_MO and US_PA data. 

It also adds the admission_type values to the dimension values validation:

ALL
LEGAL_REVOCATION
SHOCK_INCARCERATION_0_TO_6_MONTHS
SHOCK_INCARCERATION_6_MONTHS
SHOCK_INCARCERATION_9_MONTHS
SHOCK_INCARCERATION_12_MONTHS
SHOCK_INCARCERATION_PVC

The new fixture files will have removed `district` dimension from all files except the caseload file. It is also adding the fields 
`recommended_for_revocation_count` and `recommended_for_revocation_count_all` for the race and gender files.

@jovergaag 

TODO:
- [x] Get the US_MO dashboard to correctly display the district chart

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #820 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: server/collections/resources/dimensionValues/us_mo.js [hunk 1]
```diff
@@ -62,6 +63,7 @@ const DEFAULT_RISK_LEVEL = [
 ];
 
 const US_MO_DIMENSION_VALUES = {
+  admission_type: DEFAULT_ADMISSION_TYPES,
```
Comment (jovergaag) [inline server/collections/resources/dimensionValues/us_mo.js:66]: MO will only ever have 'ALL'. Admission type is a PA concept.

File: server/core/demo_data/supervision_location_ids_to_names.json [hunk 2]
```diff
@@ -25,3 +25,99 @@
...
 {"state_code":"US_PA","level_3_supervision_location_external_id":"WR","level_3_supervision_location_name":"WESTERN","level_2_supervision_location_external_id":"09","level_2_supervision_location_name":"Altoona DO","level_1_supervision_location_external_id":"09 - ALTOONA DO","level_1_supervision_location_name":"09 - Altoona DO"}
 {"state_code":"US_PA","level_3_supervision_location_external_id":"WR","level_3_supervision_location_name":"WESTERN","level_2_supervision_location_external_id":"CO","level_2_supervision_location_name":"Central Office","level_1_supervision_location_external_id":"CO - CENTRAL OFFICE","level_1_supervision_location_name":"CO - Central Office"}
+{"state_code":"US_MO","level_3_supervision_location_external_id":"NOT_APPLICABLE","level_3_supervision_location_name":"NOT_APPLICABLE","level_2_supervision_location_external_id":"TCSTL","level_2_supervision_location_name":"TCSTL","level_1_supervision_location_external_id":"SLCRC","level_1_supervision_location_name":"St. Louis Community Release Center"}
```
Comment (jovergaag) [inline server/core/demo_data/supervision_location_ids_to_names.json:28]: This was smart to just add the MO rows. And let the DistrictsStore filter out the rows from the other tenant. Great idea.

File: server/core/demo_data/revocations_matrix_filtered_caseload.metadata.json [hunk 3]
```diff
@@ -1 +1 @@
... "2382019", "2382040", "2382137", "2382233", "2382611", "2382800", "2383347", "2384329", "2384739", "2385014", "2385034", "2385420", "238570", "2385973", "2386029", "2386066", "2386327", "2386427", "2387064", "2387170", "2387463", "2387650", "2387749", "2388502", "2388588", "2388718", "2389109", "2389329", "2389370", "2389602", "2390059", "2390438", "2390649", "2390926", "2391120", "2391134", "239134", "2391464", "2392000", "2392268", "2392303", "2392508", "2392694", "2393196", "2393419", "2393664", "2394218", "2394366", "2394912", "2395190", "2395337", "2395628", "2395934", "2396473", "2396613", "2397245", "2397337", "2398711", "2398770", "2398877", "2398969", "2399902", "2399958", "2399987", "2400496", "2400840", "2400874", "2400907", "2401456", "2401478", "2401773", "2401833", "2401916", "2402074", "2402224", "2402523", "2402680", "2402784", "2402985", "2403087", "2403348", "2403548", "2403974", "2404089", "2404277", "2404283", "2404637", "2405310", "2406183", "240620", "2406713", "2407175", "2407296", "2408357", "2409701", "2410761", "2410800", "2411225", "2411465", "2411470", "2411484", "2411827", "2412257", "2412258", "2413042", "2413098", "2413442", "2414055", "2414302", "24
```
Comment (jovergaag) [inline server/core/demo_data/revocations_matrix_filtered_caseload.metadata.json:1]: Were you going to remove `district` from this file in this PR so we don't have to touch the fixture files again?

**Review by jovergaag — COMMENTED** [review]
Question about the caseload file. LMK if you want to do it in this PR or if we just follow up with an adjustment. The adjustment is easier because we just remove the column from these already randomized files so it is a much smaller lift.

I'm leaning toward fixing case table and changing fixture in a different issue/PR but if you want to tackle it here go ahead.

**Comment by daschi** [comment]
@jovergaag I would rather tackle it in a different PR since there will be a bit of FE work that will need to go along with it!

File: server/collections/resources/dimensionValues/us_pa.js [hunk 4]
```diff
@@ -52,6 +62,7 @@ const DEFAULT_CHARGE_CATEGORIES = ["all"];
 const DEFAULT_RISK_LEVEL = ["high", "low", "medium", "not_assessed"];
 
 const US_PA_DIMENSION_VALUES = {
+  admission_type: DEFAULT_ADMISSION_TYPES,
```
Comment (jovergaag) [inline server/collections/resources/dimensionValues/us_pa.js:65]: I believe that these validations should still pass with the `admission_type` dimension now added. We only fail validations if we have an extra value. Does that sound right to you? Have you deployed this to staging and tested the cache refresh with this updated validation?

Comment (daschi) [reply server/collections/resources/dimensionValues/us_pa.js:65]: Yeah these should still pass. It only fails if the dimension exists in the fetched metadata and it includes a value that is not listed in the valid dimension values.

**Review by jovergaag — APPROVED** [review]

**Pull request merged**